<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFriends</class>
 <widget class="QWidget" name="FormFriends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>536</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ebrima</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SteamAchievementsStatistic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="GroupBoxFilter">
     <property name="title">
      <string>     Фильтр</string>
     </property>
     <layout class="QVBoxLayout" name="_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="LineEditName">
          <property name="placeholderText">
           <string>Найти друга</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonFind">
          <property name="text">
           <string>  Поиск</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="ComboBoxStatus">
          <property name="currentText">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckBoxOpenProfile">
          <property name="text">
           <string>Публичный профиль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckBoxFavorites">
          <property name="text">
           <string>Избранные</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="TableFriends"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
